proto.h

```diff
@@ -19,6 +19,8 @@
 #ifndef RDESKTOP_PROTO_H
 #define RDESKTOP_PROTO_H
 
+#include <netdb.h>
```

Comment:
A small optimisation is to just declare `struct addrinfo;` here. You don't need the full header here.

Reply:
Will make it a forward declaration